src/js/views/renderCharacter.js

```diff
@@ -32,22 +32,22 @@ export const createHeroListElement = () => {
 };
 
 // eslint-disable-next-line max-len
-export const insertCharacterHTML = (heroNode, heroName, heroIntelligence, heroStrength, heroCombat) => {
-  heroNode.insertAdjacentHTML('beforeend', characterHTML(heroName, heroIntelligence, heroStrength, heroCombat));
+export const insertCharacterHTML = (heroNode, heroName, heroIntelligence, heroStrength, heroCombat, herosImage) => {
+  heroNode.insertAdjacentHTML('beforeend', characterHTML(heroName, heroIntelligence, heroStrength, heroCombat, herosImage));
 };
 
 export default function renderCharacters(arr) {
 // eslint-disable-next-line no-unused-vars
-  arr.forEach((element) => {
-    console.log(element);
+  arr.forEach(async (element) => {
```

Comment:
I'm curious about this `async` -- I don't know enough about how it works, so I would check with another dev to see if this is necessary.

With regards to naming stuff: when you are declaring variables you can call them whatever you want, so it might be nice to be more descriptive! Instead of `arr` you could call that array `characters`, and then as you loop through the array you could call each individual element `character` instead of `element`. Just a suggestion, it wouldn't change how anything is working.

Reply:
I'm going to go in and test with/without async. I will also rename those inputs so they are more descriptive (thanks for that reminder!) I will fix, add to the PR and re-request review. :)

Reply:
Also Ben, yes your explanation of when to squash makes sense. Also *what* to squash in this case--also makes sense. I agree about squashing the changes to renderCharacter and then for the renderChracterTest. I will squash those before rebasing/merging to main. Thanks!